docs/pages/development/helper.rst

```diff
@@ -0,0 +1,75 @@
+CLI Helper
```

Comment:
Since this is well-documented (as opposed to a temporary hack that we don't want to advertise), we should probably put something about it in the changelog. You can link to this PR

Reply:
I thought I added one for that, will have a look.